src/test/resources/regressions/features/loops/loops-break2.gobra

```diff
@@ -0,0 +1,23 @@
+// Any copyright is dedicated to the Public Domain.
+// http://creativecommons.org/publicdomain/zero/1.0/
+
+package pkg
+
+func foo() {
+	var a [15]int
+	invariant 0 <= i && i <= cap(a)
+	for i := 0; i < cap(a); i++ {
+		a[i] = i
+	}
+	var contains5 = false
+	invariant 0 <= i && i <= cap(a)
+	invariant contains5 == (exists j int :: 0 <= j && j < i && a[j] == 5)
```

Comment:
I think it would be good habit to have proper triggers for all testcases (for demonstration purposes but also to avoid performance issues)
```suggestion
	invariant contains5 == (exists j int :: { a[j] } 0 <= j && j < i && a[j] == 5)
```